pkg/controller/eks/fargateprofile/controller.go

```diff
@@ -37,7 +37,7 @@ import (
 	"github.com/crossplane/crossplane-runtime/pkg/reconciler/managed"
 	"github.com/crossplane/crossplane-runtime/pkg/resource"
 
-	"github.com/crossplane/provider-aws/apis/eks/v1alpha1"
+	v1alpha1 "github.com/crossplane/provider-aws/apis/eks/manualv1alpha1"
```

Comment:
can we use here `manualv1alpha1` without `v1alpha1` and change the code like in identityproviderconfig

`"github.com/crossplane/provider-aws/apis/eks/manualv1alpha1"`

Reply:
Fixed